spec/haskell/src/SEL4/Model/StateData/AARCH64.hs

```diff
@@ -13,6 +13,7 @@ module SEL4.Model.StateData.AARCH64 where
 import Prelude hiding (Word)
 import SEL4.Machine
 import SEL4.Machine.Hardware.AARCH64 (PTE(..), PT_Type, config_ARM_PA_SIZE_BITS_40)
+import SEL4.Object.Structures
```

Comment:
I'm noticing some import hierarchy changes. TCB has some anti-circular-import magic. Just checking: do your changes to the Haskell compile?

Reply:
Yeah, that anti-circular-import magic was a pain to relearn and get right. I've tested building it, it does compile.